pyproject.toml

```diff
@@ -6,10 +6,11 @@ readme = "README.md"
 requires-python = ">=3.10"
 dependencies = [
     "apscheduler>=3.10.4",
-    "duckdb>=1.1.3",
+    "duckdb==1.1.2",
```

Comment:
We should force duckdb to consider the `images` field as a string so that we're not bound to this version of duckdb (and it's safer for us).

Reply:
Actually the field is considered as a MAP, which is a STRUCT with more flexibility in key naming
Version 1.0.0 considers JSON as string during conversion, and versions > 1.0.0 consider it as MAP. I feel this is a path duckdb decided to take. Pyarrow supports MAP, which is the lib behind duckdb for parquet, and provide [features](https://arrow.apache.org/docs/python/generated/pyarrow.MapType.html) to support it. 
Thus, I propose to keep the images as MAP instead of string